src/main/java/mServer/crawler/sender/hr/HrSendungOverviewCallable.java

```diff
@@ -0,0 +1,58 @@
+package mServer.crawler.sender.hr;
+
+import de.mediathekview.mlib.Config;
+import de.mediathekview.mlib.daten.DatenFilm;
+import de.mediathekview.mlib.daten.ListeFilme;
+import de.mediathekview.mlib.tool.Log;
+import java.io.IOException;
+import java.util.List;
+import java.util.concurrent.Callable;
+import mServer.crawler.CrawlerTool;
+import static mServer.crawler.sender.MediathekReader.urlExists;
+
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+
+public class HrSendungOverviewCallable implements Callable<ListeFilme> {
+
+    private final HrSendungenDto dto;
+    private final HrSendungOverviewDeserializer overviewDeserializer = new HrSendungOverviewDeserializer();
+    private final HrSendungDeserializer sendungDeserializer = new HrSendungDeserializer();
+    
+    public HrSendungOverviewCallable(HrSendungenDto aDto) {
+        dto = aDto;
+    }
+    
+    @Override
+    public ListeFilme call() throws Exception {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 18 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)